### PR TITLE
Prevent app banners from appearing unstyled and FOUC

### LINKED ISSUE #10910: App banners causes flash of unstyled content
This is because the CSS isn't inlined. Instead, the CSS arrives later and the styles are applied in a subsequent paint.

### PR DESCRIPTION
**Pull request by OliverJAsh**
Fixes #10910

This is a race condition: if the CSS loads before the JS executes, you don't get a flash, otherwise you do.

One solution would be to add this CSS to the inline CSS, but that would increase the inefficiency and bloat of our inline CSS, as we hardly ever show these banners.

This solution will force the JS to wait until the CSS has loaded before injecting the DOM element.

I wonder if we could move more CSS out of inline and towards this model of waiting to execute JS.

/cc @sndrs

### CODE REVIEW

File: common/app/templates/headerInlineJS/config.scala.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ window.guardian = {
             )
     ),
     css: {
-        loaded: false
```
Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/config.scala.js]: This wasn't used, but now it is :smile:

File: common/app/templates/headerInlineJS/config.scala.js [hunk 2]
```diff
@@ -18,7 +18,8 @@ window.guardian = {
...
+        loaded: false,
+        loadedListeners: []
```
Comment (sndrs) [inline common/app/templates/headerInlineJS/config.scala.js]: I'd call this `loadQueue` or something similar, since they're not listening, they're called

Comment (OliverJAsh) [inline common/app/templates/headerInlineJS/config.scala.js]: Done

**Comment by sndrs** [comment]
:+1:

**Pull request merged**